Dockerfile

```diff
@@ -26,6 +26,7 @@ COPY . .
 # FIXME(vdemeester) change from docker-app to dev once buildkit is merged in moby/docker
 FROM dev AS cross
 RUN make cross
+RUN BIN_NAME=yamlschema make cross
```

Comment:
I don't believe this is needed as the `Makefile` seems to take care of it?

Reply:
Well, let me fix the build and then clean all this mess 😅